editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -5730,8 +5731,16 @@ void SpatialEditorPlugin::snap_cursor_to_plane(const Plane &p_plane) {
 	spatial_editor->snap_cursor_to_plane(p_plane);
 }
 
-void SpatialEditor::register_gizmo_plugin(Ref<EditorSpatialGizmoPlugin> ref) {
+void SpatialEditor::add_gizmo_plugin(Ref<EditorSpatialGizmoPlugin> ref) {
```

Comment:
Should be `p_ref` according to our convention, though IMO `p_plugin` might be clearer.